explorer/websockethandlers.go

```diff
@@ -96,10 +116,41 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 					}
 
 				case "getmempooltxs":
+					exp.pageData.RLock()
 					exp.MempoolData.RLock()
-					msg, err := json.Marshal(exp.MempoolData)
+
+					mempoolVotes := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Votes)
+					mempoolTickets := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Tickets)
+					mempoolRevs := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Revocations)
+					mempoolTxs := exp.blockData.GetTrimmedMempoolTx(exp.MempoolData.Transactions)
+
+					// calculate total fees for mempool block
+					getTotalFee := func(txs []*TrimmedTxInfo) (total float64) {
+						for _, tx := range txs {
+							total += tx.Fees
+						}
+						return
+					}
+					mempoolFees := getTotalFee(mempoolTxs) + getTotalFee(mempoolRevs) + getTotalFee(mempoolTickets) +
```

Comment:
I'm still quite uneasy about adding a large number of RPCs (getrawtransaction verbose) under the control of the websocket client.  It seems to me that just like the MempoolInfo struct, the MempoolData should be *updated* in `(exp *explorerUI) mempoolMonitor`, when there is a mempool event.  I say updated because there is no reason to repeatedly call getrawtransaction for the same transactions over and over.

This change must be made for this to be OK.